exp/runtime/internal/controllers/extensionconfig_controller.go

```diff
@@ -163,3 +201,41 @@ func discoverExtensionConfig(ctx context.Context, runtimeClient runtimeclient.Cl
 	conditions.MarkTrue(discoveredExtension, runtimev1.RuntimeExtensionDiscoveredCondition)
 	return discoveredExtension, nil
 }
+
+// reconcileCABundle reconciles the CA bundle for the ExtensionConfig.
+// cert-manager code: pkg/controller/cainjector/sources.go certificateDataSource (avoid cert-manager import via unstructured).
```

Comment:
```suggestion
// cert-manager code: pkg/controller/cainjector/sources.go certificateDataSource
```

the last part was only a comment to myself while we still considered also resolving Certificates (vs only Secrets what we know want to do)